src/components/base/button/index.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import propTypes from "prop-types";
+
+import Styles from "./index.module.scss";
+
+function Button({ children, onButtonClick, className, buttonType }) {
```

Comment:
It increases the readability of the code if we use more meaningful names while naming variables. Like we could have written `buttonLabel` or `buttonTitle` instead of `children` here.